### PR TITLE
Update as of Day #80

### PR DESCRIPTION
**Pull request by Bannerman**

### CODE REVIEW

File: Divi 100/README.md [hunk 1]
```diff
@@ -52,3 +52,32 @@ Every single day, for 100 days in a row, we will be sharing a new and exciting f
...
+75. [5 Beautiful Ways to Style Divi’s Shop Module](http://www.elegantthemes.com/blog/divi-resources/5-beautiful-ways-to-style-divis-shop-module)
+76. [5 Creative Divi Header Module Styles You Can Achieve Using Built-In Settings](http://www.elegantthemes.com/blog/divi-resources/5-creative-divi-header-module-styles-you-can-achieve-using-built-in-settings)
+77. [Free Divi Landing Pages Layout Pack Kit (Wireframe) Vol. 1](http://www.elegantthemes.com/blog/divi-resources/download-the-free-divi-landing-pages-layout-pack-built-with-wireframe-kit-vol-1)
+78. [How to Give Your Divi Archive Pages a Masonry Layout](http://www.elegantthemes.com/blog/divi-resources/how-to-give-your-divi-archive-pages-a-masonry-layout)
+79. [Free Divi Wireframe Product Sections Layout Kit](http://www.elegantthemes.com/blog/divi-resources/download-the-free-divi-wireframe-static-product-sections-layout-kit)
+80. [Divi 3.0 Sneak Peek Video #2](http://www.elegantthemes.com/blog/divi-resources/divi-3-0-sneak-peek-video-2-the-invisible-customizable-user-interface?utm_source=Elegant+Themes&utm_campaign=2e26172f7f-Divi_100_Daily&utm_medium=email&utm_term=0_c886a2fc0a-2e26172f7f-50751429)
```
Comment (andyhqtran) [inline Divi 100/README.md]: This should be removed from the link to make it cleaner.

> ?utm_source=Elegant+Themes&utm_campaign=2e26172f7f-Divi_100_Daily&utm_medium=email&utm_term=0_c886a2fc0a-2e26172f7f-50751429

**Comment by andyhqtran** [comment]
Awesome thanks for the PR! @Bannerman 😄

File: Divi 100/README.md (hunk 1 above)
Comment (Bannerman) [inline Divi 100/README.md]: Fixed! Didn't see that.

**Comment by Bannerman** [comment]
np! Can't wait to see 3.0!

**Pull request merged**